jsons/Policies.json

```diff
@@ -947,6 +955,8 @@
                 "uniques": [
                     "Free [Great Scientist] appears",
                     "Free [Great Engineer] appears",
+                    "[+20]% Production when constructing [Spaceship part] buildings [in all cities]",
+                    //can't add "Spaceflight Pioneers: May finish Spaceship parts with Great Engineers"
```

Comment:
Should be a request for a new unique in Unciv. Will reject addressing like this here